test/support/supportedExtensions.ts

```diff
@@ -0,0 +1,32 @@
+/* tslint:disable max-line-length */
+import { languages } from '../../src/icon-manifest/languages';
+import {
+  IExtensionCollection,
+  IFileExtension,
+  FileFormat,
+  ISpecialFileCollection,
+  SpecialExtensionType,
+} from '../../src/models';
+
+export const extensions: (IExtensionCollection<IFileExtension> & ISpecialFileCollection) = {
+  special: {
```

Comment:
Why go the extra mile?

I believe a simple structure like 

```
export const extensions: IExtensionCollection<IFileExtension> = {
   defaultl: {},
   supported: {}
}
```
would suffice.